doc/topics/venafi/venafi.rst

```diff
@@ -13,66 +13,57 @@ Open up ``/etc/salt/master`` and add:
 
 .. code-block:: yaml
 
-    api_key: None
+    venafi:
+      api_key: None
 
-Then register your email address with Venagi using the following command:
+Then register your email address with Venafi using the following command:
 
 .. code-block:: bash
 
-    salt-run venefi.register <youremail@yourdomain.com>
+    salt-run venafi.register <youremail@yourdomain.com>
 
 This command will not return an ``api_key`` to you; that will be send to you
 via email from Venafi. Once you have received that key, open up your ``master``
 file and set the ``api_key`` to it:
 
 .. code-block:: yaml
 
-    api_key: abcdef01-2345-6789-abcd-ef0123456789
-Before using these modules you need to register an account with Venafi, and
-configure it in your ``master`` configuration file.
-
-First, you need to add a placeholder to the ``master`` file. This is because
-the module will not load unless it finds an ``api_key`` setting, valid or not.
-Open up ``/etc/salt/master`` and add:
-
-.. code-block:: yaml
-
-    api_key: None
-
-Then register your email address with Venagi using the following command:
-
-.. code-block:: bash
-
-    salt-run venefi.register <youremail@yourdomain.com>
+    venafi:
+      api_key: abcdef01-2345-6789-abcd-ef0123456789
 
-This command will not return an ``api_key`` to you; that will be send to you
-via email from Venafi. Once you have received that key, open up your ``master``
-file and set the ``api_key`` to it:
-
-.. code-block:: yaml
-
-    api_key: abcdef01-2345-6789-abcd-ef0123456789
+Example Usage
+~~~~~~~~~~~~~~~~
+Generate a CSR and submit it to Venafi for issuance, using the 'Internet' zone:
+salt-run venafi.request minion.example.com minion.example.com zone=Internet
 
+Retrieve a certificate for a previously submitted request with request ID
+aaa-bbb-ccc-dddd:
+salt-run venafi.pickup aaa-bbb-ccc-dddd
 
 Runner Functions
 ~~~~~~~~~~~~~~~~
 
-gen_rsa
+gen_key
 -------
-Generate and return an RSA ``private_key``. If a ``dns_name`` is passed in, the
+Generate and return a ``private_key``. If a ``dns_name`` is passed in, the
 ``private_key`` will be cached under that name. 
 
+The key will be generated based on the policy values that were configured
+by the Venafi administrator. A default Certificate Use Policy is associated
+with a zone; the key type and key length parameters associated with this value
+will be used.
+
 :param str minion_id: Required. The name of the minion which hosts the domain
     name in question.
 
 :param str dns_name: Required. The FQDN of the domain that will be hosted on
     the minion.
 
-:param str zone: Optional. Default value is "default". The zone on Venafi that
+:param str zone: Required. Default value is "default". The zone on Venafi that
```

Comment:
Actually we changed the default zone names to uppercase; so let's make the default value the string 'Default'.